bin/setup-oscar-login-page.sh

```diff
@@ -21,5 +21,5 @@ fi
 cp docker/oscar/conf/templates/AcceptableUseAgreement.txt ./volumes/OscarDocument/login/AcceptableUseAgreement.txt
 
 # Address and Link settings from the local.env file.
-sed -e "s|supportLink=.*|supportLink=${SUPPORT_LINK//&/\\&}|g" -e "s|supportText=.*|supportText=${SUPPORT_TEXT//&/\\&}|g" -e "s|supportName=.*|supportName=${SUPPORT_NAME//&/\\&}|g" -e "s|clinicLink=.*|clinicLink=${CLINIC_LINK//&/\\&}|g" -e "s|clinicText=.*|clinicText=${CLINIC_TEXT//&/\\&}|g" -e "s|clinicName=.*|clinicName=${CLINIC_NAME//&/\\&}|g" -e "s|tabName=.*|tabName=${TAB_NAME//&/\\&}|g" docker/oscar/conf/templates/login_page_template.txt > login_page_template.txt.tmp  
+sed -e "s|supportLink=.*|supportLink=${SUPPORT_LINK//&/\\&}|g" -e "s|supportText=.*|supportText=${SUPPORT_TEXT//&/\\&}|g" -e "s|supportName=.*|supportName=${SUPPORT_NAME//&/\\&}|g" -e "s|clinicLink=.*|clinicLink=${CLINIC_LINK//&/\\&}|g" -e "s\clinicText=.*\clinicText=${CLINIC_TEXT//&/\\&}\g" -e "s|clinicName=.*|clinicName=${CLINIC_NAME//&/\\&}|g" -e "s|tabName=.*|tabName=${TAB_NAME//&/\\&}|g" docker/oscar/conf/templates/login_page_template.txt > login_page_template.txt.tmp  
```

Comment:
Change here clinicText regex character from using | to \ to prevent from error when setting up login page. \ is most likely unused so it's used